docs/integrations/api.md

```diff
@@ -20,13 +20,13 @@ https://eth-api.lido.fi/v1/protocol/steth/apr/sma
 
 Response schema and examples are available in the [Swagger API documentation](https://eth-api.lido.fi/api/static/index.html#/APR%20for%20Eth%20and%20stEth/ProtocolController_findSmaAPRforSTETH)
```

Comment:
seems like the link is broken

Reply:
Thank you! Fixed.
The Swagger URL was updated this morning.